js/pokemon.js

```diff
@@ -83,8 +105,12 @@ $(document).ready(function() {
 
     loadState();
     
-    generateNewNumbers(true);
```

Comment:
This generateNewNumbers didn't seem to have any function, as it was being called again in newPokemon anyway. Double check I didn't break anything if you get a chance.